docs/software/system_stats.md

```diff
@@ -197,8 +197,12 @@ Netdata is a slick and feature-rich system stats monitor, with web interface.
     `/etc/netdata/netdata.conf`  
     and change the line  
     `bind socket to IP = 127.0.0.1`  
-    to match either the local network IP or static public IP of your server depending on your needs.  
-    Alternatively comment it e.g. if your server does not have a static public IP but you require remote access.
+    to match either the local network IP or static public IP of your dietpi server depending on your needs.
+    Alternatively comment it e.g. if your server does not have a static public IP but you require remote access. (Probably not a good idea. See "Security hardening" in next tab.)
```

Comment:
While the note in the parentheses is a good one, IMO it looks to much like what it is, a patch that seems to counteract the remaining infos. Why would we first tell anyone to comment the IP binding when we don't think that it's a good idea 😄. So let's try to bring those together smoothly. Suggestion:
```
Alternatively comment it e.g. if your server does not have a static public IP but you require remote access. But note that an unprotected publicly accessible Netdata web interface is a potential security risk. Head over to the "Security hardening" tab to learn more about how to restrict access to Netdata.
```

Reply:
I like it. I clicked "resolve conversation" which I hope has the affect of accepting this.